components/Footer/Footer.tsx

```diff
@@ -55,70 +54,21 @@ const Footer = ({
-          205.807L15.4527 206.73L19.4145 234.344L44.1611 230.794ZM63.0354 112.257L40.6409 101.145L28.2197
-          126.177L54.4744 135.746L63.0354 112.257ZM337.152 22.7429L356.599 38.4524L375.793 14.6916L348.707
-          0.573594L337.152 22.7429ZM423.106 108.699L445.275 97.1446L431.158 70.0583L407.397 89.2512L423.106
-          108.699ZM351.455 401.711C422.483 363.439 470.846 288.322 470.846 201.871H420.846C420.846 269.208
-          383.225 327.796 327.737 357.695L351.455 401.711ZM283.382 367.622C298.249 367.622 311.074 376.608
-          316.615 389.545L362.577 369.861C349.441 339.189 318.959 317.622 283.382 317.622V367.622ZM247.265
-          403.053C247.629 383.428 263.663 367.622 283.382 367.622V317.622C236.36 317.622 198.144 355.302
-          197.273 402.124L247.265 403.053ZM19.4145 234.344C34.0502 336.356 116.468 416.42 219.611
-          427.447L224.926 377.73C144.604 369.143 80.3074 306.701 68.9077 227.243L19.4145 234.344ZM78.2273
-          169.703C78.2273 189.242 62.7067 205.166 43.3345 205.807L44.9878 255.78C91.226 254.25 128.227
-          216.303 128.227 169.703H78.2273ZM54.4744 135.746C68.3692 140.81 78.2273 154.137 78.2273
-          169.703H128.227C128.227 132.459 104.599 100.798 71.5963 88.7689L54.4744 135.746ZM243.975
-          -25C154.813 -25 77.7115 26.4384 40.6409 101.145L85.4298 123.37C114.384 65.0203 174.529 25
-          243.975 25V-25ZM348.707 0.573594C317.349 -15.7712 281.703 -25 243.975 -25V25C273.467 25
-          301.204 32.1982 325.596 44.9121L348.707 0.573594ZM348.601 61.1242C348.601 52.5088 351.586
-          44.6585 356.599 38.4524L317.704 7.03334C305.767 21.8107 298.601 40.6647 298.601
-          61.1242H348.601ZM384.726 97.2484C364.775 97.2484 348.601 81.0751 348.601 61.1242H298.601C298.601
-          108.689 337.161 147.248 384.726 147.248V97.2484ZM407.397 89.2512C401.191 94.2641 393.341 97.2484
-          384.726 97.2484V147.248C405.184 147.248 424.038 140.083 438.815 128.147L407.397 89.2512ZM470.846
-          201.871C470.846 164.145 461.618 128.502 445.275 97.1446L400.936 120.254C413.649 144.645 420.846
-          172.381 420.846 201.871H470.846ZM259.997 201.073C259.997 209.919 252.826 217.091 243.98
-          217.091V267.091C280.44 267.091 309.997 237.534 309.997 201.073H259.997ZM243.98 185.055C252.826
-          185.055 259.997 192.227 259.997 201.073H309.997C309.997 164.613 280.44 135.055 243.98
-          135.055V185.055ZM227.962 201.073C227.962 192.227 235.133 185.055 243.98 185.055V135.055C207.519
-          135.055 177.962 164.613 177.962 201.073H227.962ZM243.98 217.091C235.133 217.091 227.962 209.919
-          227.962 201.073H177.962C177.962 237.534 207.519 267.091 243.98 267.091V217.091Z" fill='#3A86FF'
-					fillOpacity="0.9" mask="url(#path-4-inside-1_387_2)" />
-				</svg>
-				<h3 className="footer-title">
-          StudyCrew
+				<img src="/assets/Logo.svg" alt="Logo" className='w-10'/>
```

Comment:
Can you please use the `next/image` component for this? It should look like this:

```javascript
import Image from 'next/image'
import Logo from '../../public/assets/Logo.svg'

<Image alt='Logo' src={Logo} className='w-10' />
```